include/dump_offload.hpp

```diff
@@ -283,6 +283,12 @@ inline void requestRoutes(App& app)
             std::string dumpType = "bmc";
             boost::asio::io_context* ioCon = conn.getIoContext();
 
+            std::filesystem::path dir(unixSocketPathDir);
+            if (!std::filesystem::exists(dir))
+            {
+                std::filesystem::create_directories(unixSocketPathDir);
```

Comment:
Please see if you can use this util method to create directories and set permissions to it https://github.com/ibm-openbmc/bmcweb/blob/1020/include/ibm/utils.hpp#L13